### DISCUSSION TITLE
Su'aal

### DISCUSSION THREAD

**Discussion by mucaadciiltire** [sharafdin/markdown-master#6] [Q&A] [answered]
extension ka luqadee adeegsateen maxaana u adeegsateen?

**Comment by omartood** [comment, marked as answer]
Waxan adegsaney Typescript, Sawabto an u adegsaneyna waa vscode ayaa ku dhisan marka waxad adegsan karta javascript ama typescripy,anaga lawadooda ba wa adegsany laakin luuqaada uu ka kacaayo ayaa ah typescript, balse javascript meelo qaar ayan u adegsaney marka waa sidas pro.

**Reply by mucaadciiltire** [reply]
waa mhdsnthy wll
